osmosis-1/osmosis-1.chainlist.json

```diff
@@ -5548,7 +5548,7 @@
           }
         ]
       },
-      "description": "Composable is the base layer connecting L1s and L2s. We are scaling IBC to other ecosystems, pushing the boundaries of trust-minimized interoperability. We abstract the cross-chain experience for users, delivering seamless chain-agnostic execution of user intentions.",
```

Comment:
the Chainlist should not be edited directly. Any changes of data should be done at the Chain Registry, where the automated script pulls this data from